libvast/src/system/index.cpp

```diff
@@ -239,6 +239,7 @@ index_state::status(status_verbosity v) const {
       // Hence the fallback to low-level primitives.
       layout_object.insert_or_assign(name, std::move(xs));
     }
+    put(stats_object, "meta-idx-bytes", meta_idx.size());
```

Comment:
The changelog says `meta-index-bytes`, this says `meta-idx-bytes`. I prefer the former.